src/components/ProjectImportModal/index.tsx

```diff
@@ -25,14 +25,19 @@ interface ProjectImportModalProps {
   onSubmit: (data?: ProjectFormValues, success?: boolean) => void
 }
 
+// 내부에서만 사용할 확장된 저장소 인터페이스
+interface EnhancedRepository extends Repository {
+  repoId: number // 실제 GitHub 저장소 ID
+}
+
 /**
  * 프로젝트 정보 입력을 위한 모달 컴포넌트
  */
 export const ProjectImportModal = ({
   onClose,
   onSubmit,
 }: ProjectImportModalProps) => {
-  const [repositories, setRepositories] = useState<Repository[]>([])
+  const [repositories, setRepositories] = useState<EnhancedRepository[]>([])
   const [submitStatus, setSubmitStatus] = useState<
     'idle' | 'loading' | 'error'
   >('idle')
```

Comment:
submitStatus와 errorMessage를 처리를 하나의 useState로 관리하는 건 어떨까요?

```suggestion
  const [submitState, setSubmitState] = useState<{
    status: 'idle' | 'loading' | 'error'
    message?: string
  }>({ status: 'idle' })
```

이렇게 바꾸면 상태와 에러메시지를 따로 업데이트 하지 않고 한 줄로 처리 가능!

```
setSubmitState({ status: 'error', message: '오류 메시지' })
```